### PR TITLE
Merge To Main

### PR DESCRIPTION
**Pull request by abdallahmalima**
In this milestone I:
- Create a Basic structure and install dependencies for react and redux
- Create Components for Financial Statements and Income statements
- Create a slice for  Financial Statements and Income statements
- Create thunk for Call API for Financial Statements and Income statements
- Add Test for Financial Statements and Income statements

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,209 @@
-# Financial-Statements
-This app helps users to familiarize themselves with financial statements and stock information for various companies. Additionally, it provides a convenient way for individuals to stay informed about the financial health and performance of different businesses.
+
+<!-- TABLE OF CONTENTS -->
```
Comment (MeqdamAlqudah) [inline README.md:2]: - [ ] Please record a video presenting the features of the project.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,209 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- Thanks to the Microverse team.
+- Thanks to the Code Reviewer(s) for the insightful feedback.
+- Hat tip to anyone whose code was used.
+
```
Comment (MeqdamAlqudah) [inline README.md:192]: - [ ] Please give appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

**Review by MeqdamAlqudah — CHANGES_REQUESTED** [review]
Hi @abdallahmalima , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
:+1: use of redux
:+1: design

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (rloterh) [reply README.md:192]: well resolved! ✔️

File: src/index.css [hunk 3]
```diff
@@ -0,0 +1,21 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
```
Comment (rloterh) [inline src/index.css:4]: - [ ] OPTIONAL
The styling of the list on your Homepage does not conform to the alternating dual-color pattern of the provided [design template](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)). Kindly note we as much as possible want your design to match the provided [template](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)). Therefore kindly ensure that the cells/ card of your list follow the **alternating dual-color pattern**. Each cell of the list item should have surrounding cells having **different (alternating) background color(s)**.

**Your design:**        |  **The original design:**
:-------------------------:|:-------------------------:
Home Page | Home Page

File: README.md [hunk 4]
```diff
@@ -1,2 +1,206 @@
...
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://github.com/abdallahmalima/Financial-Statements/blob/dev/README.md) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (rloterh) [inline README.md:206]: - [ ] OPTIONAL
The provided MIT license link in your ReadMe is invalid, kindly endeavor to fix it, and remove the guide note below it.

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Antony,  

Your project is complete! 🏁 Awesome job! There is nothing else to say other than... it's time to merge it 🍾 Congratulations!  🎉💯🌟 

## Highlights 👍
- Nice presentation of this project!
- Good job acknowledging Nelson Sakwa on Behance for the [Original design idea](https://www.behance.net/sakwadesignstudio)!
- API fetch properly implemented!
- State management with Redux properly implemented!
- Navigation working as intended!
- Filter works great!
- Good ReadMe!

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

_Kindly consider my inline comments marked **[OPTIONAL]** under this review_

**You can also consider:**
- You used incorrect git-flow/ GitHub flow for this project. Kindly note that capstone project worked on by a single contributor should have merge request flow **from feature into development**. Kindly note this important step for subsequent projects.
 
Keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**